### PR TITLE
Migrate to conda 3  and bump v223

### LINKED ISSUE #8: New home URL
According to PyPI this lives in [this repo]( https://github.com/mhammond/pywin32 ) now.

### PR DESCRIPTION
**Pull request by carlodri**
Fixes #8, closes #7

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-  {{ hash_type }}: {{ hash }}
+  url: https://github.com/mhammond/{{ name }}/archive/b{{ version }}.tar.gz
+  sha256: c8be831cb0fc11badf27b6edc0da08984322452e5c98ac84aa158cc89f6d0837
 
 build:
   number: 0
   script:
-    - python setup.py install --record=record.txt  # [not py3k]
-    - python setup3.py install --record=record.txt  # [py3k]
+    - "{{ PYTHON }} setup.py install --record=record.txt"  # [not py3k]
```
[the remark this answers is not in the document]
Comment (carlodri) [reply recipe/meta.yaml]: hi @nicoddemus, thanks for stepping in! I guess it's safer not to use `pip`given that they explicitly state that it's only experimentally supported.

[the remark this answers is not in the document]
Comment (nicoddemus) [reply recipe/meta.yaml]: No problem, although I've been using in all the other recipes I maintain without problems. Either way, I understand `pywin32` is a little tricky to get working, so no worries! 😁

**Comment by nicoddemus** [comment]
It seems the tests are failing because `CONDA_PY` is always `36`. One solution would be to move the check for the DLLs to `run_test.py` and use `sys.version_info` to generate the correct filename.

**Comment by carlodri** [comment]
thanks for the suggestion @nicoddemus, I've launched an attempt for that solution, let's see. Anyway, this could fix the `py35` build, but the `py27` apparently has another problem.

> It seems the tests are failing because CONDA_PY is always 36

And wouldn't this be a conda-something bug?

**Comment by nicoddemus** [comment]
> And wouldn't this be a conda-something bug?

It makes sense, I'm not sure...

**Comment by carlodri** [comment]
ok, looks like the tests in `py35` are fixed, thanks for the suggestion. Any ideas on how to tackle the `py27` failure?

**Comment by nicoddemus** [comment]
My guess is that AppVeyor installs the redist somewhere else that is not what is defined in `setup.py`.

**Comment by nicoddemus** [comment]
I pushed some code to find if it is in an alternate location

**Comment by carlodri** [comment]
yikes! look like the file is not there....

what about https://github.com/AnacondaRecipes/pywin32-feedstock/tree/master/recipe? Can we get any hint from there?

**Comment by nicoddemus** [comment]
Nothing jumps at me by looking at the recipe and patches, but it doesn't hurt to try to copy the recipe verbatim and see how it goes. Can you take the time to do it @carlodri?

**Comment by carlodri** [comment]
I can try, I'll need some time since there are also a number of patches to transfer.

I'm giving it another shot before that by changing the compiler specs from `cxx` to `c`.

**Comment by carlodri** [comment]
well, now the error changed and is quite mysterious to me:

```
error: [Errno 2] No such file or directory: 'build\\temp.win-amd64-2.7\\Release\\win32/src/PerfMon\\perfmondata.dll.manifest'
The system cannot find the path specified.
```

**Comment by nicoddemus** [comment]
Hmmm probably need to patch this part as well: https://github.com/mhammond/pywin32/blob/484aa6464dcb05d909ac63bfc43c39a59925367f/setup.py#L231-L238

It seems we can remove the `os.path.basename(output_filename).startswith("perfmondata.dll")` line, if the file is not being generated for some reason.

Pinging the anaconda recipe maintainers to see if they have some suggestions: @msarahan @nehaljwani @mingwandroid

**Comment by nicoddemus** [comment]
@carlodri I created #10 to track the issue with py27. I suggest for now we skip `py27` and if someone steps in to fix the problem we can build a new package for py27 then. What do you think?

**Comment by carlodri** [comment]
> @carlodri I created #10 to track the issue with py27. I suggest for now we skip py27 and if someone steps in to fix the problem we can build a new package for py27 then. What do you think?

I agree, this at least will release the `py3k` builds, which probably (maybe) will satisfy most of the users.

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Pull request merged**